docs/ml-toolkit/example.md

```diff
@@ -38,6 +46,13 @@ a spiky model response which has mainly two reasons:
       points of parameters (here temperature) are likely to be similar in 
       the Earth system context in their spatio-temporal vicinity.
 
+<p align="center">
+<img src="../img/train_test_assignment_rnd.png" width="70%" height="70%">
+</p>
+<p align = "center"><i>
+Random Assignment of Train/Test Split</i>
+</p>
+
 ### Block Sampling
 
 It is therefore mandatory to enable machine learning that respects the basic 
```

Comment:
I don't understand why the word "therefore" is used.

Reply:
It assigns to the problems mentioned before (see Random Sampling)